code/modules/mob/living/simple_animal/hostile/ordeal/steel.dm

```diff
@@ -66,35 +74,44 @@
 	maxHealth = 750
 	health = 750
 	rapid_melee = 2
-	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 1, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 0.8)
-	dextrous = FALSE
+	move_to_delay = 2.5
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.6, WHITE_DAMAGE = 1, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 0.8)
 	attack_verb_continuous = "slashes"
 	attack_verb_simple = "slash"
 	deathsound = 'sound/voice/mook_death.ogg'
-	butcher_results = list(/obj/item/food/meat/slab/human = 1, /obj/item/food/meat/slab/human/mutant/moth = 2)
+	butcher_results = list(/obj/item/food/meat/slab/human = 2, /obj/item/food/meat/slab/human/mutant/moth = 2)
 
-/mob/living/simple_animal/hostile/ordeal/steel_dawn/steel_noon/MeleeAction()
-	health+=10
+	var/self_destruct_delay = 2 SECONDS
+	var/self_destruct_radius = 2
+	var/self_destruct_damage = 50
+
+/mob/living/simple_animal/hostile/ordeal/steel/noon/AttackingTarget()
+	adjustBruteLoss(-10)
 	if(health <= maxHealth*0.15 && stat != DEAD && prob(75))
-		walk_to(src, 0)
+		move_to_delay += 4
```

Comment:
May activate multiple times because nothing here stops them from calling "AttackingTarget()" while exploding, so they could very rapidly hit double digit move_to_delay.